laboratory/src/org/labkey/laboratory/LaboratoryServiceImpl.java

```diff
@@ -308,9 +309,9 @@ public void registerClientDependency(ClientDependency cd, Module owner)
         _clientDependencies.put(owner, list);
     }
 
-    public Set<ClientDependency> getRegisteredClientDependencies(Container c)
+    public Set<Supplier<ClientDependency>> getRegisteredClientDependencies(Container c)
     {
-        Set<ClientDependency> set = new HashSet<>();
+        Set<Supplier<ClientDependency>> set = new HashSet<>();
```

Comment:
Use a List instead? Does this care about order?

Reply:
I think the original code was actually wrong here - it should have preserved order, though apparently it didnt matter much.

I agree w/ Adam, I think this method should return a List, since that's how Supplier<ClientDependency> is passed around now.